index.html

```diff
@@ -37,46 +38,265 @@
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+
+        <div>
+          <h4 class="height_inc">LET'S CONNECT</h4>
+          <ul class="small_text">
+            <li>
+              <a href="">
+                <img class="size" src="./Image/twitter.svg" alt="" />
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <img class="size" src="./Image/linkedin.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="">
+                <img class="size" src="./Image/medium.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="">
+                <img class="size" src="./Image/github.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href="">
+                <img class="size" src="./Image/angellist.svg" alt="" /></a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <a class="see_projects" href="">Get my Resume</a>
+        </div>
+      </div>
+      <div class="grid-item grid-item-2 flex-container">
+        <div class="flex-item flex-item-1 p-1 category-header">
+          <h3 class="header-3">Languages</h3>
+          <img src="./work_img/arrow.svg" class="arrow down-arrow" alt="down-arrow" />
+        </div>
+        <div class="flex-item flex-item-2">
+          <ul class="big_circle">
+            <li class="">
+              <img src="./work_img/java.svg" alt="JavaScript" />
+              <span><a href="#"> JavaScript</a></span>
+            </li>
+            <li>
+              <img src="./work_img/html.svg" alt="HTML" /><span>
+                <a href="#"> HTML</a></span>
+            </li>
+            <li>
+              <img src="./work_img/css.svg" alt="CSS" />
+              <span><a href="#"> CSS</a></span>
+            </li>
+          </ul>
+        </div>
+        <div class="flex-item flex-item-3 p-1 category-header bt-1">
+          <h3 class="header-3">Frameworks</h3>
+          <img src="./work_img/arrow_right.svg" class="arrow right-arrow" alt="down-arrow" />
+        </div>
+        <div class="flex-item flex-item-4 p-1 category-header bt-1">
+          <h3 class="header-3">Skills</h3>
+          <img src="./work_img/arrow_right.svg" class="arrow right-arrow" alt="down-arrow" />
+        </div>
+      </div>
+    </div>
+  </article>
+  <article class="contact">
+    <div class="grid-container">
+      <div class="grid-item grid-item-1 flex-container contact_flex">
+        <h2 class="contact_header flex-item-1">Contact Me</h2>
+        <p class="contact_paragraph flex-item-2">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I’d love
+          to help with it
```

Comment:
- [ ] As the previous reviewer suggested, please stick to the Figma design as much as possible. Let's add a background image from the .css file for this section
### Your Design
![image](https://user-images.githubusercontent.com/46839548/165401427-e5d8db12-59e8-4329-a129-5941826faa99.png)

### Figma Design
![image](https://user-images.githubusercontent.com/34578631/165385802-977ad3ac-13e7-4c9a-b960-ab5b8af26f33.png)